### PR TITLE
Add MPC-HC

### PR DESCRIPTION
**Pull request by vincecima**

### CODE REVIEW

File: just-install.json [hunk 1]
```diff
@@ -391,6 +391,14 @@
       },
       "version": "5.0.4"
     },
+    "media-player-classic-home-cinema": {
```
Comment (lvillani) [inline just-install.json]: I was wondering that maybe we can use `mpc-hc` here since it looks like the software is mostly known by this name.

Comment (vincecima) [inline just-install.json]: Will do :+1:

Comment (vincecima) [inline just-install.json]: Worth documenting the naming convention somewhere?

**Comment by vincecima** [comment]
@lvillani all set

File: just-install.json [hunk 2]
```diff
@@ -432,6 +432,14 @@
       },
       "version": "4.0.2"
     },
+    "mpc-hc": {
+      "installer": {
+        "kind": "innosetup",
+        "x86": "https://binaries.mpc-hc.org/MPC%20HomeCinema%20-%20Win32/MPC-HC_v1.7.10_x86/MPC-HC.1.7.10.x86.exe",
```
Comment (lvillani) [inline just-install.json]: Can you replace `1.7.10` with `{{.version}}` for both x86 and x86_64 URLs?

**Comment by lvillani** [comment]
Thanks! :+1: (I'll do the change from 1.7.10 to `{{.version}}` myself).

**Pull request merged**